src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.jsx';
+
+const ChatLog = (props) => {
```

Comment:
@ellenjin, is there a reason why you didn't de-structure your props here? Only asking out of curiosity!